src/client/ace/mode-mips.js

```diff
@@ -51,8 +53,10 @@ ace.define("ace/mode/mips",["require","exports","module","ace/lib/oop","ace/mode
                 next: "string"
             }, {
                 // directive
-                token: 'directive.keyword.control',
-                regex: /[.]\S*/
+                token: function(directive, whitespace) {
+                    return ['directive.keyword.control', ''];
+                },
+                regex: /([.]\S+)(\s+)/
```

Comment:
actually those changes are not required anymore, as we use custom regex for parsing the code

but maybe they will be used later again (cf. #73) so I'd vote for keeping it